src/Scripts/UI/Views/ProjectMenuActions.gd

```diff
@@ -187,22 +187,24 @@ func _on_fdg_open_project_file_selected(path):
 
 func _on_open_recent_project_selected(project_name: String):
 	var project_path = SingletonObject.get_project_path(project_name)
-	open_project_given_path(project_path)
+	var status = open_project_given_path(project_path)
+	if status == 0:
+		SingletonObject.ErrorDisplay("Project file no found", "the project was not found at the path it was saved. \n Maybe it was moved or deleted")
 
 
-func open_project_given_path(project_path: String):
+func open_project_given_path(project_path: String) -> int:
 	#SingletonObject.show_loading_screen("loading project...")
 	var proj_file = FileAccess.open(project_path, FileAccess.READ)
 	
 	if proj_file == null:
-		push_error("Couldn't parse the project file at %s. Error code: %s" % [project_path, FileAccess.get_open_error()])
-		return
+		push_error("Couldn't parse the proj	ect file at %s. Error code: %s" % [project_path, FileAccess.get_open_error()])
+		return 0
 	
 	var json = JSON.parse_string(proj_file.get_as_text())
 	
 	if json == null:
 		push_error("Couldn't parse the project file at %s" % project_path)
-		return
+		return 0
```

Comment:
No need to fix right now, but in future you could probably use built-in Godot error codes.
Eg. `return OK` or `return ERR_DOES_NOT_EXIST`